docs/UserGuide.adoc

```diff
@@ -46,100 +47,210 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
 ****
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit patient 1 c/91234567` +
+Edits the phone number of the 1st patient to be `91234567`.
+* `edit donor 2 n/Betsy Crower b/A` +
+Edits the name and blood type of the 2nd donor to be `Betsy Crower` and `A` respectively.
 
 === Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Finds persons whose attributes contain any of the given keywords. +
+Format: `find [patient/donor/doctor] [attributes]`
+
+A list of patient/donor/doctor's information whose attributes partially match the keywords +
+will be displayed.
 
 ****
 * The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Persons matching at least one keyword will be returned (i.e. `OR` search). +
+e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 ****
 
 Examples:
 
-* `find John` +
-Returns `john` and `John Doe`
-* `find Betsy Tim John` +
-Returns any person having names `Betsy`, `Tim`, or `John`
+* `find patient n/Lim` +
+Returns any patient whose name contains 'Lim'
+
 
 // tag::delete[]
 === Deleting a person : `delete`
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX`
+Deletes the specified person from OTM. +
+Format: `delete [patient/doctor/donor] **NRIC**`
+
+Example:
+
+`delete patient S98654322D` +
+Deletes a patient with the ID S98654322D in OTM.
+
+// end::delete[]
+=== Detecting matches: `match`
+
+There are three types of matches:
+
+* Blood type match
+* Tissue match
+* Cross match
+
+Displays a list of potential matches between donors' and patients' kidneys.
 
 ****
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index *must be a positive integer* 1, 2, 3, ...
+* A patient and donor who does not pass all three tests will be listed as potential matches.
+* A patient and donor who passes all three tests will be listed as a match.
+* `match cross` will only run matches on patients and donors whose blood type and tissue matches.
+* For each match, the success rate will be computed according to all three test results.
 ****
 
-Examples:
+Format: `match [blood/tissue/cross]`
```

Comment:
btw i think we agreed on `matchall` command to return all patients with just the numbers of matches, then `match` is for displaying a particular persons' matches, like `match ic/12341231` will display all donors that match that guy. Then we process it, `process pat/G123123 don/A1234123` will process the patient and donor